app/classifier/tasks/generic-editor.cjsx

```diff
@@ -7,6 +7,34 @@ alert = require '../../lib/alert'
 DrawingTaskDetailsEditor = require './drawing-task-details-editor'
 NextTaskSelector = require './next-task-selector'
 
+DrawingToolsEditor = React.createClass
```

Comment:
Why's this piece its own module? Can't we just just show or hide the `<option>`?

If it's got to be its own module, can we include all the drawing task stuff?